modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -712,13 +723,21 @@ export default class GPUGridAggregator {
 
   getAggregationModel(fp64 = false) {
     const {gl} = this;
+
+    const shaders = mergeShaders(
+      {
+        vs: fp64 ? AGGREGATE_TO_GRID_VS_FP64 : AGGREGATE_TO_GRID_VS,
```

Comment:
Drop legacy `fp64` support

Reply:
I had issues enabling `fp64` extension, I have todo item in the issue.
Aggregation only needs `fp64` module not the `project64`, will cleanup this code in a seperate PR.